### PR TITLE
Advanced automation hook

### PR DESCRIPTION
**Pull request by sgupta3112**
Hi @rcvrgs, 

I have raised this pull request for AdvancedPlayback sample app hook. 

Please review the code and merge it to master. 

If any changes are required then please let me know.

### CODE REVIEW

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m [hunk 1]
```diff
@@ -91,11 +92,34 @@ - (void)addAllPlayerSelectionOptions {
 - (void)viewDidLoad {
   [super viewDidLoad];
   self.navigationController.navigationBar.translucent = NO;
-  [self.tableView registerNib:[UINib nibWithNibName:@"TableCell" bundle:nil]forCellReuseIdentifier:@"TableCell"];
+    
+  
+    UISwitch *swtLog = [[UISwitch alloc] init];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m]: please use the same indentation we have in the rest of the code.

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m [hunk 2]
```diff
@@ -91,11 +92,34 @@ - (void)addAllPlayerSelectionOptions {
...
+    UIBarButtonItem * lbl = [[UIBarButtonItem alloc] initWithCustomView:lblLog];
+    self.navigationItem.rightBarButtonItems = [NSArray arrayWithObjects:btn,lbl, nil] ;
+    
+    [self.tableView registerNib:[UINib nibWithNibName:@"TableCell" bundle:nil]
+     forCellReuseIdentifier:@"TableCell"];
 
   [self addAllPlayerSelectionOptions];
 }
 
+     - (void)changeSwitch:(id)sender{
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m]: indentation too

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m [hunk 3]
```diff
@@ -91,11 +92,34 @@ - (void)addAllPlayerSelectionOptions {
...
+         if([sender isOn]){
+             NSLog(@"Switch is ON");
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m]: remove the NSLogs

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m [hunk 4]
```diff
@@ -91,11 +92,34 @@ - (void)addAllPlayerSelectionOptions {
...
+             self.qaLogEnabled=YES;
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m]: use spaces to separate the = sign and the declarations, e.g. `self.qaLogEnabled = YES;`

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m [hunk 5]
```diff
@@ -58,6 +60,8 @@ - (IBAction)onRightBtnClick:(UIButton *)sender
 
 - (void)viewDidLoad {
   [super viewDidLoad];
+    
+    appDel = [[UIApplication sharedApplication] delegate];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m]: indentation here too

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m [hunk 6]
```diff
@@ -8,23 +8,25 @@
...
-- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption {
-  self = [super initWithPlayerSelectionOption: playerSelectionOption];
+{
+ AppDelegate *appDel;
+}
+- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption qaModeEnabled:(BOOL)qaModeEnabled {
+  self = [super initWithPlayerSelectionOption: playerSelectionOption qaModeEnabled:qaModeEnabled];
   self.nib = @"PlayerDoubleButton";
-
+NSLog(@"value of qa mode in ChangeVideoProgrammaticallyViewController %@", self.qaModeEnabled ? @"YES" : @"NO");
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m]: remove this NSLog

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m [hunk 7]
```diff
@@ -70,6 +74,14 @@ - (void)viewDidLoad {
                                            selector:@selector(notificationHandler:)
                                                name:nil
                                              object:_ooyalaPlayerViewController.player];
+    
+    // In QA Mode , making textView visible
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m]: identation too

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m [hunk 8]
```diff
@@ -88,10 +100,23 @@ - (void) notificationHandler:(NSNotification*) notification {
     return;
   }
 
-  NSLog(@"Notification Received: %@. state: %@. playhead: %f",
-        [notification name],
-        [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
-        [self.ooyalaPlayerViewController.player playheadTime]);
+    NSString *message = [NSString stringWithFormat:@"Notification Received: %@. state: %@. playhead: %f count: %d",
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m]: indentation too

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m [hunk 9]
```diff
@@ -88,10 +100,23 @@ - (void) notificationHandler:(NSNotification*) notification {
...
+                         [notification name],
+                         [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
+                         [self.ooyalaPlayerViewController.player playheadTime], appDel.count];
+    
+    NSLog(@"%@",message);
+    
+    //In QA Mode , adding notifications to the TextView
+    if(self.qaModeEnabled==YES) {
+        NSString *string = self.textView.text;
+        NSString *appendString = [NSString stringWithFormat:@"%@ :::::::::: %@",string,message];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/ChangeVideoPlayerViewController.m]: spacing after the comma, make it more readable like `..., string, message];`

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m [hunk 10]
```diff
@@ -26,12 +27,16 @@ @interface CustomControlsPlayerViewController ()
...
+}
 
-- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption {
-  self = [super initWithPlayerSelectionOption: playerSelectionOption];
+- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption qaModeEnabled:(BOOL)qaModeEnabled {
+  self = [super initWithPlayerSelectionOption: playerSelectionOption qaModeEnabled:qaModeEnabled];
   self.nib = @"PlayerSimple";
 
-  if (self.playerSelectionOption) {
+     NSLog(@"value of qa mode in CustomControlPlayerViewController %@", self.qaModeEnabled ? @"YES" : @"NO");
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m]: remove the NSLog

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m [hunk 11]
```diff
@@ -49,6 +54,8 @@ - (void)loadView {
 
 - (void)viewDidLoad {
   [super viewDidLoad];
+    
+    appDel = [[UIApplication sharedApplication] delegate];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m]: identation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m [hunk 12]
```diff
@@ -61,6 +68,13 @@ - (void)viewDidLoad {
                                            selector:@selector(notificationHandler:)
                                                name:nil
                                              object:_ooyalaPlayerViewController.player];
+    
+    // In QA Mode , making textView visible
+    if(self.qaModeEnabled==YES){
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m [hunk 13]
```diff
@@ -80,10 +94,21 @@ - (void) notificationHandler:(NSNotification*) notification {
     return;
   }
 
-  NSLog(@"Notification Received: %@. state: %@. playhead: %f",
-        [notification name],
-        [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
-        [self.ooyalaPlayerViewController.player playheadTime]);
+    NSString *message = [NSString stringWithFormat:@"Notification Received: %@. state: %@. playhead: %f count: %d",
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m [hunk 14]
```diff
@@ -80,10 +94,21 @@ - (void) notificationHandler:(NSNotification*) notification {
...
+                         [notification name],
+                         [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
+                         [self.ooyalaPlayerViewController.player playheadTime], appDel.count];
+    
+    NSLog(@"%@",message);
+    
+    //In QA Mode , adding notifications to the TextView
+    if(self.qaModeEnabled==YES) {
+        NSString *string = self.textView.text;
+        NSString *appendString = [NSString stringWithFormat:@"%@ :::::::::: %@",string,message];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomControlsPlayerViewController.m]: comma spacing

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomOverlayPlayerViewController.m [hunk 15]
```diff
@@ -28,11 +29,12 @@ @interface CustomOverlayPlayerViewController ()
...
 @implementation CustomOverlayPlayerViewController
+  AppDelegate *appDel;
 
-- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption {
-  self = [super initWithPlayerSelectionOption: playerSelectionOption];
+- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption qaModeEnabled:(BOOL)qaModeEnabled {
+  self = [super initWithPlayerSelectionOption: playerSelectionOption qaModeEnabled:qaModeEnabled];
   self.nib = @"PlayerSimple";
-
+    NSLog(@"value of qa mode in CustomOverlayPlayerViewController %@", self.qaModeEnabled ? @"YES" : @"NO");
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomOverlayPlayerViewController.m]: remove NSLog

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomOverlayPlayerViewController.m [hunk 16]
```diff
@@ -53,6 +55,8 @@ - (void)loadView {
 
 - (void)viewDidLoad {
   [super viewDidLoad];
+    
+    appDel = [[UIApplication sharedApplication] delegate];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomOverlayPlayerViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomOverlayPlayerViewController.m [hunk 17]
```diff
@@ -65,6 +69,12 @@ - (void)viewDidLoad {
                                            selector:@selector(notificationHandler:)
                                                name:nil
                                              object:_ooyalaPlayerViewController.player];
+    
+    // In QA Mode , making textView visible
+    if(self.qaModeEnabled==YES){
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomOverlayPlayerViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomOverlayPlayerViewController.m [hunk 18]
```diff
@@ -83,10 +93,23 @@ - (void) notificationHandler:(NSNotification*) notification {
     return;
   }
 
-  NSLog(@"Notification Received: %@. state: %@. playhead: %f",
-        [notification name],
-        [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
-        [self.ooyalaPlayerViewController.player playheadTime]);
+    NSString *message = [NSString stringWithFormat:@"Notification Received: %@. state: %@. playhead: %f count: %d",
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/CustomOverlayPlayerViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m [hunk 19]
```diff
@@ -21,10 +22,11 @@ @interface InsertAdPlayerViewController ()
 @end
 
 @implementation InsertAdPlayerViewController
-
-- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption {
-  self = [super initWithPlayerSelectionOption: playerSelectionOption];
+ AppDelegate *appDel;
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m]: Add braces as the rest of the files. e.g. 
```
 @implementation InsertAdPlayerViewController {
  AppDelegate *appDel;
}
```

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m [hunk 20]
```diff
@@ -21,10 +22,11 @@ @interface InsertAdPlayerViewController ()
...
+- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption qaModeEnabled:(BOOL)qaModeEnabled{
+  self = [super initWithPlayerSelectionOption: playerSelectionOption qaModeEnabled:qaModeEnabled];
   self.nib = @"PlayerDoubleButton";
+     NSLog(@"value of qa mode in InsertAdPlayerViewController %@", self.qaModeEnabled ? @"YES" : @"NO");
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m]: remove NSLog

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m [hunk 21]
```diff
@@ -58,13 +60,19 @@ - (IBAction)onRightBtnClick:(UIButton *)sender
...
-  
+  appDel = [[UIApplication sharedApplication] delegate];
   [self.button1 setTitle:@"INSERT VAST AD" forState:UIControlStateNormal];
   [self.button2 setTitle:@"INSERT OOYALA AD" forState:UIControlStateNormal];
 
   // Create Ooyala ViewController
   OOOoyalaPlayer *player = [[OOOoyalaPlayer alloc] initWithPcode:self.pcode domain:[[OOPlayerDomain alloc] initWithString:self.playerDomain]];
   self.ooyalaPlayerViewController = [[OOOoyalaPlayerViewController alloc] initWithPlayer:player];
+    // In QA Mode , making textView visible
+    if(self.qaModeEnabled==YES){
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m [hunk 22]
```diff
@@ -91,10 +99,23 @@ - (void) notificationHandler:(NSNotification*) notification {
     return;
   }
 
-  NSLog(@"Notification Received: %@. state: %@. playhead: %f",
-        [notification name],
-        [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
-        [self.ooyalaPlayerViewController.player playheadTime]);
+    NSString *message = [NSString stringWithFormat:@"Notification Received: %@. state: %@. playhead: %f count: %d",
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m [hunk 23]
```diff
@@ -91,10 +99,23 @@ - (void) notificationHandler:(NSNotification*) notification {
...
+                         [notification name],
+                         [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
+                         [self.ooyalaPlayerViewController.player playheadTime], appDel.count];
+    
+    NSLog(@"%@",message);
+    
+    //In QA Mode , adding notifications to the TextView
+    if(self.qaModeEnabled==YES) {
+        NSString *string = self.textView.text;
+        NSString *appendString = [NSString stringWithFormat:@"%@ :::::::::: %@",string,message];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m]: spacing after comma

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/LocalizationLanguagesViewController.m [hunk 24]
```diff
@@ -19,12 +20,13 @@ @interface LocalizationLanguagesViewController ()
...
-- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption {
-  self = [super initWithPlayerSelectionOption: playerSelectionOption];
+@implementation LocalizationLanguagesViewController{
+  AppDelegate *appDel;
+}
+- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption qaModeEnabled:(BOOL)qaModeEnabled{
+  self = [super initWithPlayerSelectionOption: playerSelectionOption qaModeEnabled:qaModeEnabled];
   self.nib = @"PlayerSimple";
-  
+   NSLog(@"value of qa mode in LozalizationLanguageViewController %@", self.qaModeEnabled ? @"YES" : @"NO");
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/LocalizationLanguagesViewController.m]: remove NSLog

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/LocalizationLanguagesViewController.m [hunk 25]
```diff
@@ -53,6 +57,12 @@ - (void)viewDidLoad {
                                            selector:@selector(notificationHandler:)
                                                name:nil
                                              object:self.ooyalaPlayerViewController.player];
+    
+    // In QA Mode , making textView visible
+    if(self.qaModeEnabled==YES){
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/LocalizationLanguagesViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/LocalizationLanguagesViewController.m [hunk 26]
```diff
@@ -89,11 +99,22 @@ - (void) notificationHandler:(NSNotification*) notification {
     return;
   }
   
-  NSLog(@"Notification Received: %@. state: %@. playhead: %f",
-        [notification name],
-        [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
-        [self.ooyalaPlayerViewController.player playheadTime]);
+    NSString *message = [NSString stringWithFormat:@"Notification Received: %@. state: %@. playhead: %f count: %d",
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/LocalizationLanguagesViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/LocalizationLanguagesViewController.m [hunk 27]
```diff
@@ -89,11 +99,22 @@ - (void) notificationHandler:(NSNotification*) notification {
...
+                         [notification name],
+                         [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
+                         [self.ooyalaPlayerViewController.player playheadTime], appDel.count];
+    
+    NSLog(@"%@",message);
+    
+    //In QA Mode , adding notifications to the TextView
+    if(self.qaModeEnabled==YES) {
+        NSString *string = self.textView.text;
+        NSString *appendString = [NSString stringWithFormat:@"%@ :::::::::: %@",string,message];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/LocalizationLanguagesViewController.m]: spacing after comma

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m [hunk 28]
```diff
@@ -20,12 +20,14 @@ @interface NotificationsPlayerViewController ()
...
+  AppDelegate *appDel;
+}
 
-- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption {
-  self = [super initWithPlayerSelectionOption: playerSelectionOption];
+- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption qaModeEnabled:(BOOL)qaModeEnabled{
+  self = [super initWithPlayerSelectionOption: playerSelectionOption qaModeEnabled:(BOOL)qaModeEnabled];
   self.nib = @"PlayerSimple";
-
+ NSLog(@"value of qa mode in NotificationPlayerViewController %@", self.qaModeEnabled ? @"YES" : @"NO");
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m]: remove NSLog

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m [hunk 29]
```diff
@@ -56,6 +58,11 @@ - (void)viewDidLoad {
                                            selector:@selector(notificationHandler:)
                                                name:nil
                                              object:self.ooyalaPlayerViewController.player];
+    
+    if(self.qaModeEnabled==YES){
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m]: identation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m [hunk 30]
```diff
@@ -45,7 +47,7 @@ - (void)loadView {
 
 - (void)viewDidLoad {
   [super viewDidLoad];
-
+appDel = [[UIApplication sharedApplication] delegate];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m [hunk 31]
```diff
@@ -208,11 +215,22 @@ - (void) notificationHandler:(NSNotification*) notification {
     NSLog(@"Note: Bitrate changed notification received: %f", self.ooyalaPlayerViewController.player.bitrate);
   }
 
-  //Generic notification handler
-  NSLog(@"Notification Received: %@. state: %@. playhead: %f count: %d",
-        [notification name],
-        [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
-        [self.ooyalaPlayerViewController.player playheadTime], ((AppDelegate *)[[UIApplication sharedApplication] delegate]).count);
-  ((AppDelegate *)[[UIApplication sharedApplication] delegate]).count++;
+    NSString *message = [NSString stringWithFormat:@"Notification Received: %@. state: %@. playhead: %f count: %d",
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m]: indentation

File: AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m [hunk 32]
```diff
@@ -208,11 +215,22 @@ - (void) notificationHandler:(NSNotification*) notification {
...
+                         [notification name],
+                         [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
+                         [self.ooyalaPlayerViewController.player playheadTime], appDel.count];
+    
+    NSLog(@"%@",message);
+    
+    //In QA Mode , adding notifications to the TextView
+    if(self.qaModeEnabled==YES) {
+        NSString *string = self.textView.text;
+        NSString *appendString = [NSString stringWithFormat:@"%@ :::::::::: %@",string,message];
```
Comment (brol1dev) [inline AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/NotificationsPlayerViewController.m]: spacing after commas

**Review by brol1dev — COMMENTED** [review]
I didn't add comments to all of the files since it was going to take a long time but I'm asking the same thing most of the times. Please try follow the same indentation we use and code conventions that you see in the app.

**Comment by sgupta3112** [comment]
Hi @rcvrgs, 

I have made all the changes as per the standards. Removed NSLogs and set the indentation. 

Please review it now and still changes are required then please let me know.

**Comment by brol1dev** [comment]
I'll merge this in, but I'll be refactoring the sample apps, including this one when I have the chance.

**Pull request merged**